index.js

```diff
@@ -4,13 +4,69 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
+header.style.color = "red"
 
 
 /***** Deliverable 3 *****/
-// console.log("the player object looks like this:", player)
+console.log("the player object looks like this:", player)
+const playerDiv = document.querySelector('.player')
+playerDiv.innerHTML = `
+<img src="${player.photo}" alt="${player.name}">
+<h2>${player.name}</h2>
+<em>${player.nickname}</em>
+<h4>Great Goals</h4>
+<ul id="goals">
+</ul>
+`
```

Comment:
This works - you can also use `querySelector` to find the elements that are already on the page and update them. Be careful with `innerHTML` when we get into the event handling code - whenever you set `innerHTML = ` to some new content, it will **replace** any existing DOM elements inside the parent element, which also means the event listeners won't work.